enterprise_access/apps/subsidy_request/management/commands/tests/test_send_admins_email_with_new_requests.py

```diff
@@ -0,0 +1,248 @@
+        call_kwargs = mock_braze_client.return_value.send_campaign_message.call_args[1]
+
+        actual_recipients = call_kwargs['recipients']
+        actual_trigger_properties = call_kwargs['trigger_properties']
+
+        assert actual_recipients[0]['attributes']['email'] == 'pieguy@example.com'
+        assert actual_recipients[1]['attributes']['email'] == 'cakeman@example.com'
+        assert actual_trigger_properties['requests'][0]['user_email'] == new_request.user.email
+        assert len(actual_trigger_properties['requests']) == 1
+
+    @mock.patch('enterprise_access.apps.subsidy_request.tasks.LmsApiClient.get_enterprise_admin_users')
+    @mock.patch('enterprise_access.apps.subsidy_request.tasks.LmsApiClient.get_enterprise_customer_data')
+    @mock.patch('enterprise_access.apps.subsidy_request.tasks.BrazeApiClient')
+    def test_new_requests_task_error(self, mock_braze_client, mock_get_ent_customer_data, mock_admin_learners):
+        """
+        Verify last_remind_date is not updated if braze email fails.
+        """
+        mock_get_ent_customer_data.return_value = {
+            'uuid': self.enterprise_customer_uuid,
+            'slug': 'test-enterprise',
+        }
+        mock_admin_learners.return_value = self.admin_learners_response
+        mock_braze_client.side_effect = HTTPError
+
+        command_name = 'send_admins_email_with_new_requests'
+
+        for _ in range(2):
+            factories.LicenseRequestFactory(
+                enterprise_customer_uuid=self.enterprise_customer_uuid,
+                state=SubsidyRequestStates.REQUESTED,
+            )
+
+        config = factories.SubsidyRequestCustomerConfigurationFactory(
+            enterprise_customer_uuid=self.enterprise_customer_uuid,
+            subsidy_requests_enabled=True,
+            last_remind_date=None
+        )
+
+        call_command(command_name)
+
+        config.refresh_from_db()
+        assert config.last_remind_date is None
+
+    @mock.patch('enterprise_access.apps.subsidy_request.tasks.LmsApiClient.get_enterprise_customer_data')
+    @mock.patch('enterprise_access.apps.subsidy_request.tasks.BrazeApiClient')
+    def test_no_new_requests(self, mock_braze_client, mock_get_ent_customer_data):
+        """
+        Verify no braze emails sent if no new requests.
+        """
+        mock_get_ent_customer_data.return_value = {
+            'uuid': self.enterprise_customer_uuid,
+            'slug': 'test-enterprise',
+        }
+
+        command_name = 'send_admins_email_with_new_requests'
+
+        for _ in range(2):
+            factories.LicenseRequestFactory(
+                enterprise_customer_uuid=self.enterprise_customer_uuid,
+                state=SubsidyRequestStates.REQUESTED,
+            )
+
+        factories.SubsidyRequestCustomerConfigurationFactory(
+            enterprise_customer_uuid=self.enterprise_customer_uuid,
+            subsidy_requests_enabled=True,
+            last_remind_date=datetime.now()
```

Comment:
should last_remind_date be None here since we want the licenses to be created after the last_remind_date?

Reply:
i think in this test we want no sub requests to be applicable 

```
past | ---> request1, r2, r3 -----> last_remind_date ------> no sub requests after ---> | future
```
^ if we run the command then there will be no requests between `last_remind_date` and the future, which is what we want here i think